### ISSUE TITLE
False positive unused `phpoffice/phpspreadsheet`

### ISSUE THREAD

**Issue by simPod** [composer-unused/composer-unused#317]
## Describe the bug

I'm using `phpoffice/phpspreadsheet` in my project. I have clear use statements as:

```php
use PhpOffice\PhpSpreadsheet\Cell\Cell;
use PhpOffice\PhpSpreadsheet\Spreadsheet;
use PhpOffice\PhpSpreadsheet\Worksheet\Worksheet;
```

`composer-unused` says it's unused.

## Additional information

v0.8.1

**Comment by icanhazstring** [comment]
Thanks. Will check on that.

**Comment by icanhazstring** [comment]
Related to: #316

**Comment by icanhazstring** [comment]
Fixed with #318
